src/Catalogue/ISBN.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akondas\Library\Catalogue;
+
+use InvalidArgumentException;
+use Isbn\Isbn as FaleIsbn;
+
+class ISBN
+{
+    private FaleIsbn $faleIsbn;
```

Comment:
:thinking: You will probably not need it as a state, you can try something like this:
```php
if(!(new FaleIsbn())->validation->isbn($isbn))
```

By the way, what does `FaleIsbn` mean? (I know what isbn is)

Reply:
Right, I will change it 👍 

"Fale" is the GitHub username of an author of the Isbn package, I use it because of names conflict. Maybe IsbnCode will be a better alias name here 🤔